test/integration/appsource/cymbal-bank/frontend/frontend_test.go

```diff
@@ -0,0 +1,114 @@
+// Copyright 2024 Google LLC
```

Comment:
I think this file should just go in `test/integration/appsource/` for the cft auto discovery?